main.py

```diff
@@ -41,6 +41,9 @@
 
 logger.info("Lemon-Pi : starting up")
 
+if not "SETTINGS_MODULE" in os.environ:
+    os.environ["SETTINGS_MODULE"] = "config.settings"
+
```

Comment:
Here we check if there has been an environment variable passed in to override the settings.
If there has not been, then we set the settings module to find the default one in config/settings.py
Notice that the "/" directory separator is turned into a "." and the suffix (.py) is not included.